### PR TITLE
remove some duplicate assertion in StringUtilsTest

### PR DESCRIPTION
**Pull request by shalk**
keep it clean

### CODE REVIEW

File: src/test/java/org/apache/commons/lang3/StringUtilsTest.java [hunk 1]
```diff
@@ -1623,13 +1622,9 @@ public void testReCapitalize() {
     public void testRemove_char() {
         // StringUtils.remove(null, *)       = null
         assertNull(StringUtils.remove(null, 'a'));
-        assertNull(StringUtils.remove(null, 'a'));
-        assertNull(StringUtils.remove(null, 'a'));
 
         // StringUtils.remove("", *)          = ""
         assertEquals("", StringUtils.remove("", 'a'));
-        assertEquals("", StringUtils.remove("", 'a'));
-        assertEquals("", StringUtils.remove("", 'a'));
```
Comment (kinow) [inline src/test/java/org/apache/commons/lang3/StringUtilsTest.java:1632]: This method, `testRemove_char`, appears to have been based off the `testRemove_String`. You are correct these three statements are duplicated and can be removed. But in `testRemove_String` there's one test case that must be added here as well:

```
assertNull(StringUtils.remove(null, null));
&
assertEquals("", StringUtils.remove("", null));
```

Can you add them, please, @shalk ?

File: src/test/java/org/apache/commons/lang3/StringUtilsTest.java [hunk 2]
```diff
@@ -2774,7 +2769,6 @@ public void testSplitPreserveAllTokens_StringChar() {
         assertEquals("b", res[1]);
         assertEquals("c", res[2]);
         assertEquals("", res[3]);
-        assertEquals("", res[3]);
```
Comment (kinow) [inline src/test/java/org/apache/commons/lang3/StringUtilsTest.java:2777]: I believe this was supposed to be `assertEquals("", res[4]);`. The other tests have a similar case, where they verify the indexes from `0` to the last index available (`4` in this case). Can you change it, please?

**Review by kinow — CHANGES_REQUESTED** [review]
There are a couple of places where I think the tests were supposed to have other values. Take a look and if you agree and have spare time, can you update them, please?

The other ones look correct to me. Great catch @shalk . Thanks for your pull request.

File: src/test/java/org/apache/commons/lang3/StringUtilsTest.java (hunk 1 above)
Comment (shalk) [reply src/test/java/org/apache/commons/lang3/StringUtilsTest.java:1632]: done

**Comment by kinow** [comment]
Merged. Thanks @shalk !!

**Pull request closed without merging**